.gitignore

```diff
@@ -0,0 +1,2 @@
+# Dependency directories
+node_modules/
```

Comment:
- [ ] [OPTIONAL] You can add your ".vscode" folder in the gitignore. Since it's just an editor settings file you don't need to track it in your repo

Reply:
Thank you, honorable code reviewer. From your valuable feedback, I just have learned a new thing. 

- Have added the ".vscode" folder in gitignore.